dashboard/src/components/Filters.js

```diff
@@ -13,21 +11,32 @@ export const filterData = (data, filters, returnWholeArray = false) => {
         .map((item, index) => {
           const itemValue = item[filter.field];
           if (['number'].includes(filter.type)) {
-            if (itemValue === filter.value) return item;
-            return null;
+            const { number, number2, comparator } = filter.value;
+            if (comparator === 'unfilled') return !itemValue || [null, undefined].includes(itemValue) ? item : null;
+            if (!itemValue || [null, undefined].includes(itemValue)) return null;
```

Comment:
ça c'est le genre de truc pas clair. On comprends pas pourquoi on a `null` et `undefined` en plus.
Falsy c'est : `null, undefined, 0, "", NaN, false`
Donc soit je n'ai pas compris et dans ce cas il faut un commentaire, soit c'est inutile et il faut enlever

```suggestion
            if (!itemValue) return null;
```